src/libs/OptionsListUtils.ts

```diff
@@ -1472,8 +1472,8 @@ function createOptionList(personalDetails: OnyxEntry<PersonalDetailsList>, repor
             }
 
             const isSelfDM = ReportUtils.isSelfDM(report);
-            // Currently, currentUser is not included in visibleChatMemberAccountIDs, so for selfDM we need to add the currentUser as participants.
-            const accountIDs = isSelfDM ? [currentUserAccountID ?? 0] : report.visibleChatMemberAccountIDs ?? [];
+            // Currently, currentUser is not included in participants, so for selfDM we need to add the currentUser as participants.
+            const accountIDs = isSelfDM ? [currentUserAccountID ?? 0] : Object.keys(report.participants ?? {}).map(Number);
```

Comment:
I think this change is causing a regression here https://github.com/Expensify/App/issues/41138.

Because `createOption` relies on `personalDetailList.length > 1`, which is not true for DM's when using `participants`, since the `participants` property has everyone in that conversation including the current user.

This also makes the above comment wrongish, for some reason the currentUser is not included as a participant in the selfDM but is included in other chats.